sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -219,39 +264,45 @@ export interface SendableMessageInfo {
     label?: string;
     messageId?: string | number | Buffer;
     partitionKey?: string;
+    properties?: {
+        [key: string]: any;
+    };
     replyTo?: string;
     replyToSessionId?: string;
     scheduledEnqueueTimeUtc?: Date;
     sessionId?: string;
     timeToLive?: number;
     to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
     viaPartitionKey?: string;
 }
 
 // @public
 export class Sender {
-    cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
-    cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
+    cancelScheduledMessage(sequenceNumber: Long, options?: OperationOptions): Promise<void>;
+    cancelScheduledMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<void>;
     close(): Promise<void>;
     get isClosed(): boolean;
-    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo): Promise<Long>;
-    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[]): Promise<Long[]>;
-    send(message: SendableMessageInfo): Promise<void>;
-    sendBatch(messages: SendableMessageInfo[]): Promise<void>;
+    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo, options?: OperationOptions): Promise<Long>;
+    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[], options?: OperationOptions): Promise<Long[]>;
+    send(message: SendableMessageInfo, options?: OperationOptions): Promise<void>;
+    sendBatch(messages: SendableMessageInfo[], options?: OperationOptions): Promise<void>;
     }
 
-// @public
+// @public (undocumented)
 export class ServiceBusClient {
     constructor(connectionString: string, options?: ServiceBusClientOptions);
-    constructor(host: string, credential: TokenCredential, options?: ServiceBusClientOptions);
-    close(): Promise<any>;
-    createQueueClient(queueName: string): QueueClient;
-    createSubscriptionClient(topicName: string, subscriptionName: string): SubscriptionClient;
-    createTopicClient(topicName: string): TopicClient;
-    readonly name: string;
+    constructor(hostName: string, tokenCredential: TokenCredential_2, options?: ServiceBusClientOptions);
+    // (undocumented)
+    close(): Promise<void>;
+    createReceiver(queueName: string, receiveMode: "peekLock"): NonSessionReceiver<"peekLock">;
+    createReceiver(queueName: string, receiveMode: "receiveAndDelete"): NonSessionReceiver<"receiveAndDelete">;
+    createReceiver(topicName: string, subscriptionName: string, receiveMode: "peekLock"): NonSessionReceiver<"peekLock"> & SubscriptionRuleManagement;
```

Comment:
I would suggest that rule management should be on the client instead of the receiver.
The current model forces one to think about receivers and receive mode when all they wanted to do was play with rules.

Reply:
@ramya-rao-a - I'm inclined to agree. I might do the same "trick" that I did with `diagnostics` and make a management sub-area.

Reply:
I will probably leave it here for preview.1 though - we didn't really talk about this across teams yet and it'd be nice to see where we can place it.

Reply:
Moved!